src/apiCalls.js

```diff
@@ -109,4 +93,10 @@ const getChatGPTRecipePitches = (allRecipes) => {
   });
 }
 
-export { currentUser, pageData, updateCurrentUser, loadData, getRecipeCard };
+export { currentUser, pageData, updateCurrentUser, loadData };
+
+
+
+
+
+
```

Comment:
a lot of extra spaces here XD

Reply:
lol had to make sure that I got that extra line at the end of the file!!

Lol jk, done now

Reply:
LOL hahaha